src/lib/fetchingUtils.js

```diff
@@ -49,6 +49,18 @@ function generateQueryUrl(serverUrl, queryObj) {
       case "id":
         urlStr += `${seperator}_id=${queryObj[param]}`;
         break;
+      case "identifier":
+        if (queryObj.system) {
+          urlStr += `${seperator}identifier=${queryObj.system}%7C${queryObj[param]}`;
+        } else {
+          urlStr += `${seperator}identifier=${queryObj[param]}`;
+        }
```

Comment:
We might also want to support scenarios where only the system is provided, the fourth option for [search tokens](https://www.hl7.org/fhir/search.html#token). I don't think we need to worry about the other type of token, where codes are provided, the `|` is used, and systems are explicitly omitted. But I can see a world where a user might type just the system in.

Reply:
Updated: I've just taken a look at the case below. I see we support that. Disregard.